### PR TITLE
Implemented fetching information about twitter relationships

### LINKED ISSUE #43: Relationships from Twitter
Fetch relationships from Twitter for users that are persisted in the Neo4j database.

### PR DESCRIPTION
**Pull request by Adebski**
Resolves #43 

For every new user we use Twitter API to download all users that follow new user. Also when a video is added to already existing user we are upadting implicit associations for all users that follow the user who rated new video.

### CODE REVIEW

File: src/main/scala/ztis/twitter/TweetProcessorActor.scala [hunk 1]
```diff
@@ -103,6 +105,12 @@ class TweetProcessorActor(tweet: Tweet,
 
         log.info(s"Persisting $toPersist")
         cassandraClient.updateRating(toPersist)
+        
+        userResponse.get.followedBy.foreach { fromUserID => 
+          val association = UserVideoImplicitAssociation(fromUserID, UserOrigin.Twitter, userID, UserOrigin.Twitter, videoID, videoOrigin)
+          cassandraClient.updateImplicitAssociation(association)
+        }
+        relationshipsFetcher.queueFetchRelationshipsFor(tweet.userId())
```
Comment (Mequrel) [inline src/main/scala/ztis/twitter/TweetProcessorActor.scala]: why do we both queue and get followers?

File: src/main/scala/ztis/util/RouterSupport.scala [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package ztis.util
+
+import akka.actor.{ActorRef, ActorSystem}
+import akka.routing.FromConfig
+
+trait RouterSupport {
```
Comment (Mequrel) [inline src/main/scala/ztis/util/RouterSupport.scala]: Like it!

**Comment by Mequrel** [comment]
LGTM

File: src/main/scala/ztis/twitter/TweetProcessorActor.scala (hunk 1 above)
Comment (Adebski) [inline src/main/scala/ztis/twitter/TweetProcessorActor.scala]: Maybe the name of the method is misleading - the idea is that you queue the request for relationships fetching. I will rename the method.

**Pull request merged**